addons/website/__manifest__.py

```diff
@@ -58,6 +58,7 @@
         'views/snippets/s_key_benefits.xml',
         'views/snippets/s_picture.xml',
```

Comment:
Commit message: the description is the same as the title, so it can be removed. Actually you can use your description as title: ```[IMP] website: introduce the `s_bento_grid` snippet```